apps/simplex-chat/Main.hs

```diff
@@ -40,11 +41,11 @@ logCfg = LogConfig {lc_file = Nothing, lc_stderr = True}
 main :: IO ()
 main = do
   ChatOpts {dbFile, smpServers} <- welcomeGetOpts
-  void $ createStore "simplex-chat.db" 4
+  st <- createStore (dbFile <> ".chat.db") 4
   ct <- newChatTerminal
-  a <- getSMPAgentClient cfg {dbFile, smpServers}
+  a <- getSMPAgentClient cfg {dbFile = dbFile <> ".agent.db", smpServers}
```

Comment:
with default db path wouldn't these become `...smp-chat.db.chat.db` and `...smp-chat.db.agent.db`? change default to `combine appDir "simplex"`?

Reply:
Yes, I didn’t check the default